chapters/operatorsandexpressions.tex

```diff
@@ -95,10 +99,10 @@ \section{Operator Precedence and Associativity}\label{operator-precedence-and-as
 The unary minus and plus in Modelica is slightly different than in Mathematica\footnote{\emph{Mathematica} is a registered trademark of Wolfram Research Inc.} and in MATLAB\footnote{\emph{MATLAB} is
 a registered trademark of MathWorks Inc.}, since the following expressions are illegal (whereas in Mathematica and in MATLAB these are valid expressions):
 \begin{lstlisting}[language=modelica]
-2*-2  // = -4 in Mathematica/MATLAB; is illegal in Modelica
---2   // = 2 in Mathematica/MATLAB; is illegal in Modelica
-++2   // = 2 in Mathematica/MATLAB; is illegal in Modelica
-2--2  // = 4 in Mathematica/MATLAB; is illegal in Modelica
+2 * -2  // = -4 in Mathematica/MATLAB; is illegal in Modelica
+- - 2   // = 2 in Mathematica/MATLAB; is illegal in Modelica
++ + 2   // = 2 in Mathematica/MATLAB; is illegal in Modelica
+2 - -2  // = 4 in Mathematica/MATLAB; is illegal in Modelica
```

Comment:
I think it looks technically correct to introduce these spaces; it doesn't change the point of the example.

I strikes me, though, that one could also have had similar examples for the operators `--` and `++` found in other languages, and for these the trick with introducing a space wouldn't have worked.

Reply:
Yes, I can open a ticket about that for LaTeXML.